src/relayclient/types/GsnTransactionDetails.ts

```diff
@@ -14,9 +14,9 @@ export default interface GsnTransactionDetails {
   gas?: PrefixedHexString
   gasPrice?: PrefixedHexString
 
-  // Required parameters for GSN:
-  readonly forwarder: Address
-  readonly paymaster: Address
+  // Required parameters for GSN, but assigned later
```

Comment:
not required per transaction... can be set in the provider

Reply:
We used to allow overrides